File: golang/1684. 统计一致字符串的数目.go
func countConsistentStrings(allowed string, words []string) (res int) {
    mask:=0
    for _,c :=range allowed{
        mask |=1<<(c-'a')
    }
    for _,word:=range words{
        mask1:=0
        for _,c :=range word{
            mask1 |=1<<(c-'a')
        }
        if mask|mask1==mask{
            res++
        }
    }
    return
}

File: cpp/1684. 统计一致字符串的数目.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int mask=0;
        for(auto c:allowed){
            mask |=1<<(c-'a');
        }
        int res=0;
        for(auto &&word:words){
            int mask1=0;
            for(auto c:word){
                mask1 |=1<<(c-'a');
            }
            if((mask | mask1)==mask){
                res++;
            }
        }
        return res;
    }
};